Demo/AuthenticatorDemo/ViewController+WordPressAuthenticator.swift

```diff
@@ -76,5 +78,24 @@ extension ViewController {
                 navTitleTextColor: .white
             )
         )
+
+        WordPressAuthenticator.shared.delegate = self
+    }
+
+    // TODO: Need to handle new user flow
```

Comment:
I need to double check this... The WordPress backend should be handling this for us already. Not sure whether it makes sense to handle creating an account in the context of the demo app.

Reply:
FWIW the demo app shouldn't have the capability to create a new user anyway – that's restricted based on the OAuth token granted to the WP/JP apps. 

If this flow allows it then sure, I guess, but I'd say for the purposes of the demo app it's ok to say "you need to have an account" instead of implementing the signup flow? 

I know this project also contains signup code, and I think for that case we could modify the back-end a little to make a public set of OAuth credentials and special-case the server to respond with something like "if this were a real app, this would've worked, but it's not so here's a real-looking response you can use to test the flow but please don't try to do anything useful with it"

Reply:
@jkmassel > FWIW the demo app shouldn't have the capability to create a new user anyway – that's restricted based on the OAuth token granted to the WP/JP apps.

With the above, do you mean that our backend should only allow creating new users with a token that has been generated using the Google apps (i.e. the client id, secret, and audience) for WordPress and Jetpack?

I hadn't considered that, but I think it makes sense. Also, I think we can safely test the new user creation in the context of Jetpack/WordPress and add a notice in the demo app that says "you should have a valid user already", as you suggested.

We can always revisit later on in case we decided to do some work around signup.

Reply:
I tweaked the presentation and method name in the demo app to make it clearer this code path only tests getting the token from Google

Reply:
> our backend should only allow creating new users with a token that has been generated using the Google apps (i.e. the client id, secret, and audience) for WordPress and Jetpack?

IINM, do the OAuth dance and get a token from Google, which we can pass to WP.com to authenticate as the given user. When we perform that request, we use the WP.com OAuth clientID/secret to authenticate as the app (but not as any user because we aren't logged in at that point). That WP.com clientID/secret is what is allowed to create new users.

Does that explain it better? 😅

Reply:
Gotcha 👍